website/i18n/en.json

```diff
@@ -37,3528 +37,6 @@
         "title": "Writing style guide",
         "sidebar_label": "Writing style guide"
       },
-      "en/packages/fast-animation/api/classes/_animate_index_.animate": {
-        "title": "en/packages/fast-animation/api/classes/_animate_index_.animate"
-      },
-      "en/packages/fast-animation/api/classes/_animatefrom_index_.default": {
```

Comment:
This looks generated - is there a reason we're tracking?

Reply:
interestingly it’s already tracked in ‘.gitignore’ so not sure whey it’s tracking.

Reply:
I think you need to remove it from your gitignore, delete the file, then re-add it to the gitignore

Reply:
This file is already omitted from the commit.  Problem solved.